packages/manager/src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworking.tsx

```diff
@@ -471,10 +468,17 @@ class LinodeNetworking extends React.Component<CombinedProps, State> {
     const sharedIPs = uniq<string>(
       pathOr([], ['ipv4', 'shared'], linodeIPs).map((i: IPAddress) => i.address)
     );
+    const ranges = linodeIPs?.ipv6?.global ?? [];
+    const sharedIPv6 = uniq<string>(
+      ranges.reduce((acc: string[], i: IPRange) => {
+        if (i.route_target !== null) {
+          return acc;
+        }
+        return acc.concat(`${i.range}/${i.prefix}`);
```

Comment:
Could this be made more concise with a ternary something like:
 ```suggestion
        if (i.route_target !== null) {
          return acc;
        }
        return acc.concat(`${i.range}/${i.prefix}`);
        return i.route_target ? acc.concat(`${i.range}/${i.prefix}`) : acc;
```